src/Raven.Studio/typescript/common/queryUtil.ts

```diff
@@ -35,7 +35,20 @@ class queryUtil {
         const escapedFieldName = queryUtil.escapeCollectionOrFieldName(fieldName);
         const escapedIndexName = queryUtil.escapeName(indexName);
         const escapedValueName = queryUtil.escapeName(value);
-        return `from index ${escapedIndexName} where ${escapedFieldName} == ${escapedValueName}`;
+        
+        const fromPart = `from index ${escapedIndexName}`;
+        let wherePart = `where ${escapedFieldName} == ${escapedValueName}`;
+        
+        if (indexName.startsWith("Auto") &&
+            ((value.startsWith("{") && (queryUtil.hasUpperCase(value))) || value === "NULL_VALUE")) {
```

Comment:
Do we need `value.startsWith("{")` condition?

Reply:
yes, to detect it is a json